noxfile.py

```diff
@@ -117,7 +118,11 @@ def set_dev(session):
     """Install PyBaMM in editable mode."""
     set_environment_variables(PYBAMM_ENV, session=session)
     envbindir = session.bin
-    session.install("-e", ".[all]", silent=False)
+    venv_dir = pathlib.Path('./venv').resolve()
+    session.install("virtualenv")
+    session.run("virtualenv", os.fsdecode(venv_dir), silent=True)
+    python = os.fsdecode(venv_dir.joinpath("bin/python"))
+    session.run(python, "-m", "pip", "install", "-e", ".[all]", silent=False)
     session.install("cmake", silent=False)
     if sys.platform == "linux" or sys.platform == "darwin":
         session.run(
```

Comment:
@arjxn-py, do you remember why we added this command? It doesn't seem to make any changes for me

Reply:
This command is setting up the environment variable `LD_LIBRARY_PATH` which is then appended to a file named `activate` located in the `.nox/dev/bin`.
Basically the significance this command holds is to activate the `venv` with `source .nox/dev/bin/activate` in GNU/Linux & MacOS & with `.nox\dev\Scripts\activate.bat` in Windows.

Reply:
@agriyakhetarpal @arjxn-py,  Should I change anything over here ?

Reply:
Given that the `pybamm-requires` session would have been run before `dev`, you can remove this command. This is because SUNDIALS would have already been set up then (`scikits.odes` would now be installed within the session because we have environment variables that point to the installation of SUNDIALS)